doc/user/content/sql/create-sink/kafka.md

```diff
@@ -332,12 +332,28 @@ Consider using the Debezium envelope if:
 ### Automatic topic creation
 
 If the specified Kafka topic does not exist, Materialize will attempt to create
-it while processing the `CREATE SINK` statement. Materialize will configure the
-topic with the broker's default number of partitions and replication factor.
-
-To customize the topic configuration, create the sink's topic outside of
-Materialize with the desired configuration (e.g., using the [`kafka-topics.sh`]
-tool) before running `CREATE SINK`.
+it using the broker's default number of partitions, default replication factor,
+default compaction policy, and default retention policy.
```

Comment:
Reworded this in the spirit of the original suggestion, but without using the passive voice.